Application/Frontend/components/Messaging/Message.tsx

```diff
@@ -26,25 +28,64 @@ import { MessageProps } from "@/accordTypes";
  * @returns {JSX.Element} A JSX element representing a single message in the chat interface, potentially
  * including an avatar, username, timestamp, and message content.
  */
-export function Message({ username, message, firstMessage, date, userProfileURL }: MessageProps) {
+export function Message({
+  id,
+  clientID,
+  username, 
+  message,
+  firstMessage,
+  date,
+  userProfileURL,
+  privateChat,
+  onMessageExchange,
+  onDeleteMessage}: DisplayedMessageProps) {
+
+  const [isHovered, setIsHovered] = useState(false);
+  
+  const theme = useComputedColorScheme();
```

Comment:
- The usage of useMantineTheme and useComputedColorScheme to determine the theme and adjust the background color based on the theme is good practice for ensuring consistency with the application's theme. However, ensure that the color calculations are accurate and that the background color contrasts well with the message content for readability.